app/src/main/java/com/goldenraven/padawanwallet/utils/AutoCompleteMnemonic.kt

```diff
@@ -0,0 +1,84 @@
+package com.goldenraven.padawanwallet.utils
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material.Text
+import androidx.compose.runtime.*
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.platform.LocalView
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.goldenraven.padawanwallet.component.SearchTextField
+import java.util.*
+
+@Composable
+fun AutoCompleteMnemonic(data:List<String>) {
+    val mnemonic = ArrayList<String>()
```

Comment:
Was there a reason to use an `ArrayList` instead of just a `List`?

Reply:
no there was no specific reason for this.

Reply:
Cool! I've been trying to use `List`, `Set`, and `Map` the most as they are the core 3 collections for Kotlin. But I'm always happy to hear if there is a particular optimization or reason to deviate from the standards.